File origsrc/libtpms-0.8.2/include/libtpms/tpm_library.h is a regular file while file src/libtpms-0.8.2/include/libtpms/tpm_library.h is a symbolic link
--- origsrc/libtpms-0.8.2/tests/Makefile.am	2021-03-02 03:51:21.000000000 +0900
+++ src/libtpms-0.8.2/tests/Makefile.am	2021-04-24 11:09:14.028117800 +0900
@@ -9,7 +9,7 @@ TESTS_ENVIRONMENT = \
   abs_top_builddir=`cd '$(top_builddir)'; pwd` \
   abs_top_srcdir=`cd '$(top_srcdir)'; pwd`
 
-AM_CFLAGS = -I$(top_srcdir)/include $(SANITIZERS)
+AM_CFLAGS = -I$(top_srcdir)/include $(SANITIZERS) -I$(top_srcdir)/include/libtpms
 AM_LDFLAGS = -ltpms -L$(top_builddir)/src/.libs $(SANITIZERS)
 
 check_PROGRAMS = \
